lib/actions/BackendAction.js

```diff
@@ -168,9 +168,13 @@ class BackendAction {
   }
 
   _pipelineChanged (file, cb) {
-    fsEx.readJson(file, {throws: false}, (err, pipeline) => {
-      if (err) return cb(err)
-      if (!pipeline || !pipeline.pipeline || !pipeline.pipeline.id) return cb(new Error('invalid pipeline'))
+    fsEx.readJson(file, {throws: true}, (err, pipeline) => {
+      if (err) {
+        const index = err.message.indexOf(':') + 2
```

Comment:
I think its better when we using [jsonlint](https://www.npmjs.com/package/jsonlint) for showing the error.
It formats the output better and show the line number